LICENSE.md

```diff
@@ -0,0 +1,11 @@
+# Released under MIT License
+
+Copyright (c) 2013 Mark Otto.
+
+Copyright (c) 2017 Andrew Fong.
```

Comment:
Don't use other people's names here. Note that maintaining the years isn't recommended by the LF and other communities: https://hynek.me/til/copyright-years/.
Also, here's an example of an MIT license in markdown: https://github.com/IQAndreas/markdown-licenses/blob/master/mit.md

Reply:
Thank you. Resolved.